src/Illuminate/Pagination/AbstractPaginator.php

```diff
@@ -306,14 +306,17 @@ public static function currentPathResolver(Closure $resolver)
 	 * Resolve the current page or return the default value.
 	 *
 	 * @param  string  $pageName
-	 * @param  int  $default
+	 * @param  int     $default
+	 * @param  int     $lastPage
 	 * @return int
 	 */
-	public static function resolveCurrentPage($pageName = 'page', $default = 1)
+	public static function resolveCurrentPage($pageName = 'page', $default = 1, $lastPage)
```

Comment:
You can't have a parameter without a default following parameters with a default.